src/gui/networksettings.cpp

```diff
@@ -15,16 +15,16 @@
 #include "networksettings.h"
 #include "ui_networksettings.h"
 
-#include "theme.h"
-#include "configfile.h"
-#include "application.h"
+#include "accountmanager.h"
+#include "clientproxy.h"
 #include "configfile.h"
 #include "folderman.h"
-#include "accountmanager.h"
+#include "theme.h"
 
+#include <QList>
 #include <QNetworkProxy>
 #include <QString>
-#include <QList>
+#include <qevent.h>
```

Comment:
please have a look

Reply:
This is on purpose, otherwise QtCore's `QEvent.h` is included, and that does not have one of the gui events.

Reply:
oO
Then please add a comment

Reply:
done